video/out/gpu/d3d11_helpers.c

```diff
@@ -201,17 +201,22 @@ static IDXGIAdapter1 *get_d3d11_adapter(struct mp_log *log,
             bstr_xappend_asprintf(NULL, listing,
                                   "Adapter %u: vendor: %u, description: %s\n",
                                   adapter_num, desc.VendorId,
-                                  adapter_description ? adapter_description : "<No Description>");
+                                  adapter_description);
         }
 
-        if (requested_adapter_name && adapter_description &&
-            !strcmp(requested_adapter_name, adapter_description))
+        if (adapter_description && requested_adapter_name.len &&
```

Comment:
The adapter_description check isn't necessary here. As above, it can never be NULL.

Reply:
Good catch.